ls1intum/Artemis
# [Hotfix] fix programming exerc dashboard not showing

### Checklist

### Motivation and Context
The Exercise Dashboard for programming Exercises does not show evaluated submissions.
[https://github.com/ls1intum/Artemis/issues/2596](url)


### Description
fixed Problem where lastResult was not correctly set in programming-submission.service

### Steps for Testing


1. Log in to Artemis
2. create a Programming Exercise in a course, with manual assessment enabled
3. Submit the the programming exercise
4. Check if it shows up on the exercise assessment dashboard as assessed submission


### Screenshots
![image](https://user-images.githubusercontent.com/33342534/102997301-07afcb80-4525-11eb-9655-c46ee025afe1.png)
